src/components/issues/Card.module.scss

```diff
@@ -22,70 +22,116 @@ $card-text: #78716c;
     }
 }
 
-.card__top {
+.card__top__details__button {
     display: flex;
-    align-items: flex-start;
-    gap: 2rem;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 0.5rem;
+    h2 {
+        font-size: 1.5rem;
+        margin: 0;
+    }
+
+    @media (max-width: 48rem) {
+        flex-direction: column;
+        align-items: center;
+    }
 }
 
-.card__top__details {
-    flex: 1;
-    max-width: 75%;
-    overflow-x: clip;
+.card__top__button {
+    @include styles.issues-button;
+    min-width: 9rem;
+    margin-left: 0.5rem;
 }
 
-.card__top__details__meta_data {
-    margin: 0 0 0.75rem;
+.card__issue_created__by,
+.card__assignee {
+    margin-bottom: 0.5rem;
 
-    & h2 {
-        font-weight: 600;
-        font-size: 1.5rem;
-        margin: 0 0 0.25rem;
-        color: #1c1917;
-    }
+    a {
+        color: styles.$link-text;
+        text-decoration: none;
+        margin-left: 0.2rem;
 
-    & p {
-        font-size: 0.875rem;
-        font-weight: 500;
-        color: $card-text;
+        &:hover {
+            text-decoration: underline;
+        }
     }
 }
 
-.card__top__details__desc {
-    color: $card-text;
-}
-
-.card__top__button {
-    @include styles.issues-button;
+.card__body__description {
+    img {
+        max-width: 100%;
+        height: auto;
+    }
 }
 
 .card__link {
-    width: 440px;
-
     font-family: 'Inter';
     font-style: normal;
-    font-weight: 500;
-    font-size: 12px;
+    font-size: 0.8rem;
     line-height: 140%;
-    /* or 17px */
     color: $card-text;
 
     & a {
         color: styles.$link-text;
+        text-decoration: none;
+
+        &:hover {
+            text-decoration: underline;
+        }
     }
 }
 
 .label {
     display: block;
     padding: 0.5rem 0.75rem;
-    font-size: 0.875rem;
+    font-size: 0.8rem;
     border: none;
     border-radius: 1.25rem;
     background: styles.$link-bg;
-    color: styles.$link-text;
     font-weight: 500;
+    font-family: 'Inter';
+    margin-right: 0.5rem;
 }
 
 .label__block {
     display: flex;
 }
+
+.card__body {
+    width: 100%;
+    color: black;
+
+    h1 {
+        font-size: 1.2rem;
+    }
+    img {
+        max-width: 100%;
+        margin: 0.5rem 0;
+        border: 1px solid $card-text;
+        border-radius: 0.5rem;
+    }
+    table {
+        width: 100%;
+        border-collapse: collapse;
+        border: 1px solid $card-text;
+        border-radius: 0.5rem;
+        margin: 0.5rem 0;
+        th,
+        td {
+            border: 1px solid $card-text;
+            padding: 0.5rem;
```

Comment:
use rem instead of px and there are many places where your have used px convert it into rem or em

Reply:
Where?